packages/hash/api/src/auth/ory-kratos.ts

```diff
@@ -0,0 +1,10 @@
+import { Configuration } from "@ory/client";
+import { V0alpha2Api as OpenSourceV0alpha2Api } from "@ory/kratos-client";
+import { getRequiredEnv } from "../util";
+
+/** @todo: connect to admin endpoint when admin methods are required */
```

Comment:
Sorry for the todo being unclear, it is related because the URL used below is the "public" endpoint of the kratos API. The kratos API also has an "admin" endpoint with a separate URL, that can be used by privileged clients (like the HASH workspace BE) to access sensitive methods (for [example](https://www.ory.sh/docs/kratos/reference/api#operation/adminCreateIdentity)).

Currently we don't have a need for the admin API, but I think we will in the next PR (for setting `shortnames` in identity schemas after a user has signed up), so I think I'll address this todo there.